### ISSUE TITLE
Watch File Ignores Parameters

### ISSUE THREAD

**Issue by adfrost81** [brion/hdrfix#3]
I've been using this with screenshots from Mass Effect Andromeda. After settling on some optimal settings, I updated the watch.bat file with them, and started taking some screenshots. Afterwards, I compared against some of my test images and they did not match. It appears as though it is ignoring whatever values are put in the batch file and just using the defaults.

Output from my settings:
![Mass Effect Andromeda Screenshot 2021 10 03 - 00 45 06 33-ActualSettings](https://user-images.githubusercontent.com/91818709/135742181-c17b90e4-3eb2-4075-bf77-d94847a91c86.jpg)

Output from watch.bat:
![Mass Effect Andromeda Screenshot 2021 10 03 - 00 45 06 33-sdr](https://user-images.githubusercontent.com/91818709/135742204-d9a8fb96-993d-4bdb-8a34-57ae4ee4ee6f.jpg)

Here's the batch file settings I used:
[watch.txt](https://github.com/brion/hdrfix/files/7272796/watch.txt)

**Comment by brion** [comment]
In general I would recommend against using `--hdr-max=70` as this will place the maximum preserved HDR brightness level below the nominal SDR white level (80 nits).

I'm also not sure I understand what the difference between "output from [your] settings" and "with watch.bat" means. How did you run it with your settings if it wasn't by running the modified watch.bat? Did you run on a specific input file with those settings but not the --watch=. setting? Did you update the right `watch.bat` file? Did you close the previously running instance of the program?

**Comment by adfrost81** [comment]
> In general I would recommend against using `--hdr-max=70` as this will place the maximum preserved HDR brightness level below the nominal SDR white level (80 nits).
> 
> I'm also not sure I understand what the difference between "output from [your] settings" and "with watch.bat" means. How did you run it with your settings if it wasn't by running the modified watch.bat? Did you run on a specific input file with those settings but not the --watch=. setting?

Correct. I experimented by passing parameters directly in the command line and observed the results. After trying many different variations, I settled on `--hr-max=70` as that is provided results closest to what I saw in the game. I presumed was giving me an HDR Max of 70% since that is what worked best for me, but I'm open to this being incorrect/wrong syntax.

>Did you update the right `watch.bat` file?

I copied the sample `watch.bat` file into the folder containing `hdrfix.exe` & the screenshots, replaced the sample settings, and saved it.

>Did you close the previously running instance of the program?

I did not have an active instance running before running the `hdrfix --watch=.` command. Perhaps my syntax is incorrect and it is ignoring the `watch.bat` file entirely?

I should also state that I tried using the default settings you provided in the `watch.bat` file just to see what would happen, and it ignored those as well.

**Comment by adfrost81** [comment]
>I presumed was giving me an HDR Max of 70% since that is what worked best for me, but I'm open to this being incorrect/wrong syntax.

Update: I verified the syntax on the `--hdr-max=70` command it is definitely not the same as `--hdr-max=70%`. That said, anything `--hdr-max=350` and higher is indistinguishable from simply not using any parameters at all. At least on my monitor (a 48" LG C1).

**Comment by adfrost81** [comment]
@brion Is it possible to run this on multiple items without using the watch file method? Like by use of a wildcard for the file names or something like that?

**Comment by adfrost81** [comment]
Closing this because I figured out what I was doing wrong, and thus confirmed it is working properly. Rather than running the batch file directly from the command line, I was using the following the following command: `>hdrfix --watch=*`

For some reason, I thought just having the batch file in the same directory as hdrfix would mean it would reference the file. Now obviously (once I thought about it more), it doesn't do that, which is why it wasn't doing anything. So I ran the `>watch.bat` and that of course worked.
